src/main/java/com/github/manolo8/darkbot/backpage/entities/galaxy/GalaxyInfo.java

```diff
@@ -1,11 +1,12 @@
 package com.github.manolo8.darkbot.backpage.entities.galaxy;
 
-import com.github.manolo8.darkbot.utils.XmlHelper;
 import org.dom4j.Element;
 
 import java.util.List;
 import java.util.stream.Collectors;
 
+import static com.github.manolo8.darkbot.utils.XmlHelper.*;
```

Comment:
avoid static imports, and avoid wildcard imports

Reply:
why 🤔

Reply:
For static imports, it's really not intuitive to see a function call without knowing it comes from another file